plugin/flags_sources/compilation_db.py

```diff
@@ -185,3 +194,29 @@ def filter_bad_arguments(argument_list):
                 continue
             new_args.append(argument)
         return new_args
+
+    def _find_related_sources(self, file_path, db):
+        if not file_path:
+            return
```

Comment:
Let's add some logging warning here for possible future debugging.

Reply:
Fixed in latest version of the branch ;-)

Reply:
Thanks!